core/operation/src/test/java/uk/gov/gchq/gaffer/operation/OperationChainTest.java

```diff
@@ -66,11 +71,11 @@ public void shouldSerialiseAndDeserialiseOperationChain() throws SerialisationEx
         assertNotNull(deserialisedOp);
         assertEquals(2, deserialisedOp.getOperations().size());
         assertEquals(OperationImpl.class, deserialisedOp.getOperations()
-                                                        .get(0)
```

Comment:
Yeah that would be useful. Not sure the best way to share that standard though. We can add notes to the ways of working wiki page. But better would be to share settings files for intellij/eclipse etc.